docs/modules/ROOT/pages/guide-metatx.adoc

```diff
@@ -0,0 +1,349 @@
+
+In the example code, the functionality of the https://github.com/OpenZeppelin/workshops/blob/master/25-defender-metatx-api/contracts/SimpleRegistry.sol[`SimpleRegistry` contract] is to take a string and store it. The contract's https://github.com/OpenZeppelin/workshops/blob/master/25-defender-metatx-api/contracts/Registry.sol[metatransaction implementation] achieves the same result by decoupling the signer from the sender of the transaction.
+
+When comparing the code, note the metatransaction's use of `_msgSender()` as opposed to the SimpleRegistry's use of `msg.sender`. By extending from `ERC2771Context` and `MinimalForwarder`, the contract becomes metatransaction capable. (Note that all OpenZeppelin contracts are compatable with the use of `_msgSender()`.)
+
+The second fundamental change between the two contracts is the need for the metatransaction contract to specify the address of the trusted forwarder, which in this case is the address of the `MinimalForwarder` contract.
+
+[[configure-project]]
+== Configure the project
+
+First, fork the repository and then `git clone` your fork to your computer and install dependencies
+
+```
+$ git clone https://github.com/[GitHub username]/workshops.git
+$ cd workshops/25-defender-metatx-api/
+$ yarn
+```
+
+Create a `.env` file in the project root and supply your Team API key and secret from Defender. A private key will be used for local testing but the Relayer is used for actual contract deployment.
+
+```
+PRIVATE_KEY="Goerli private key"
+TEAM_API_KEY="Defender Team API key, used for uploading autotask code"
+TEAM_API_SECRET="Defender Team API secret"
+```
+
+[[create-relayer]]
+== Create Relayer
+
+First, install the necessary package:
+
+`npm install defender-relay-client`
+
+Create a script such as the following, supplying your Defender API key in your .env file and adjusting variable names as appropriate.
+
+```
+const { RelayClient } = require('defender-relay-client');
+const { appendFile, readFileSync, writeFileSync} = require('fs');
+
+async function run() {
+  require('dotenv').config();
+  const { TEAM_API_KEY: apiKey, TEAM_API_SECRET: apiSecret, AUTOTASK_ID: autotaskId } = process.env;
+  const relayClient = new RelayClient({ apiKey, apiSecret });
+
+  // create relay using defender client
+  const requestParams = {
+    name: 'MetaTxRelayer',
+    network: 'goerli',
+    minBalance: BigInt(1e17).toString(),
+  };
+  const relayer = await relayClient.create(requestParams);
+  
+  // store relayer info in file
+  writeFileSync('relay.json', JSON.stringify({
+    relayer
+  }, null, 2));
+  console.log('Relayer ID: ', relayer);
+
+  // create and save the api key to .env - needed for sending tx
+  const {relay} = JSON.parse(readFileSync('relay.json'))
+  const secretKey = await relayClient.createKey(relayer.relayerId);
+  writeFileSync('.env.relay', JSON.stringify({secretKey}, null, 2));
```

Comment:
Nit: the convention for multiple env files is to separate them by environment, so this should rather be `.env.local` (see [here](https://www.prisma.io/docs/guides/development-environment/environment-variables/using-multiple-env-files)). OTOH, dotenv advises against this (see FAQ [here](https://www.npmjs.com/package/dotenv)). Also, env files should be in bash format (RELAY_KEY=foobar), not json.

Bottom line: I'd suggest you either programmatically add the relayer key to .env (`appendFileSync('.env', `RELAY_KEY=${secretKey}`), so you automatically get it in the process.env when you run require `dotenv`, or store it in a separate `key.json` file. Whichever of the two you pick, the file must be added to the gitginore.